### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewIntegratedTerminal_ScreenReader: NVDA does not read the state 

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52290]
- VSCode Version:
- OS Version:

Steps to Reproduce:

1.
2.

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by chrmarti** [comment]
Please provide more details.
